lib/game.ex

```diff
@@ -2,14 +2,38 @@ defmodule TicTacToe.Game do
   alias TicTacToe.{Board, HumanPlayer, ComputerPlayer, Validator, CliDisplay, Messager}
 
   def play_tic_tac_toe do
+    start_new_game(%{})
+  end
+
+  def start_new_game(player_scores) do
     welcome_players
     {player_1, player_2} = decide_game_type
-    start_new_game({player_1, player_2})
+    current_board = Board.create_new_board
+    player_scores = add_player_scores(player_scores, player_1, player_2)
+    take_turn(current_board, {player_1, player_2}, player_scores)
   end
 
-  def start_new_game({player_1, player_2}) do
-    current_board = Board.create_new_board
-    take_turn(current_board, {player_1, player_2})
+  def add_player_scores(player_scores, player_1, player_2) do
+    cond do
+      !Map.has_key?(player_scores, player_1.name) ->
```

Comment:
Looks like `Map.put_new/3` might be useful here:

http://elixir-lang.org/docs/stable/elixir/Map.html#put_new/3